common/content/tabs.js

```diff
@@ -49,10 +49,10 @@ const Tabs = Module("tabs", {
     },
 
     /**
-     * @property {Iterator(Object)} A genenerator that returns all browsers
+     * @return {Generator} A genenerator that returns all browsers
      *     in the current window.
      */
-    get browsers() {
+    browsers: function* () {
```

Comment:
Could have kept the getter by returning the result of an additional generator function. I am not sure which way is best.